client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+	plus as addNewSiteIcon,
+	globe as domainsIcon,
+	commentAuthorAvatar as profileIcon,
+} from '@wordpress/icons';
+import MaterialIcon from 'calypso/components/material-icon';
+import { SiteExcerptData } from 'calypso/data/sites/site-excerpt-types';
+import { navigate } from 'calypso/lib/navigate';
+import { isCustomDomain, isNotAtomicJetpack } from '../utils';
+
+export const generateCommandsArrayWpcom = ( {
+	setSelectedCommandName,
+	createSiteUrl,
+	__,
```

Comment:
I think these parameters could be declared inside the function instead of receiving them as parameters.
We may need to transform the function into a hook by prepending the word `use`.

Let's add that refactor as a follow-up PR.

Reply:
Sounds good, I will adjust in the follow up PR